ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.6.yaml

```diff
@@ -22,7 +22,12 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: VERBOSE=2 make -k test-acceptance test-acceptance-artifacts
+      commands: make -k VERBOSE=2 OPERATOR_INDEX_IMAGE_REF=$(cat /usr/local/secrets/quay.repo):pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index
```

Comment:
do we really need an extra secret to be mounted? Could we set `OPERATOR_INDEX_IMAGE_REF` to `quay.io/redhat-developer/servicebinding-operator::pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index`

Reply:
It is not really necessary, indeed. I used that to easily switch between repositories while testing the jobs with the PR images.